python/pyspark/sql/connect/session.py

```diff
@@ -10,7 +10,7 @@
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# WITHOUT WARRANTIES OR CONDITIONS OF A[attr-defined]NY KIND, either express or implied.
```

Comment:
Seems to be a mistake? Could you revert this?

Reply:
Ah thanks so much for pointing that out!